db/migrate/20160120011152_create_teams.rb

```diff
@@ -0,0 +1,12 @@
+class CreateTeams < ActiveRecord::Migration
+  def change
+    create_table :teams do |t|
+      t.string :name, null: false
+      t.string :city, null: false
```

Comment:
I think maybe city might be the wrong word. For example New England is not a city. Perhaps location?